### PR TITLE
MAINT: 2017.7 version bump (pt 1 of 2)

### PR DESCRIPTION
**Pull request by thermokarst**

### CODE REVIEW

File: urls.py [hunk 1]
```diff
@@ -61,6 +61,16 @@
     'distro/core/qiime20176-1497566713.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20176-1497566713.zip',
 
+    # 2017.7 DISTRO
+    'distro/core/qiime2-2017.7-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.6/release/qiime2-2017.7-conda-osx-64.txt',
```
Comment (jairideout) [inline urls.py]: I think these OS X and Linux install files should be using the 2017.7 staging environment files, then we can update the links in the upcoming "part 2" PR to point to release env files.

For example, https://raw.githubusercontent.com/qiime2/environment-files/master/2017.7/staging/qiime2-2017.7-conda-linux-64.txt

**Review by jairideout — CHANGES_REQUESTED** [review]
Just had one comment about the staging env files, otherwise GTG!

File: urls.py (hunk 1 above)
Comment (thermokarst) [reply urls.py]: We can actually preemptively use the release URL, which is what we have [done in the past](https://github.com/qiime2/data.qiime2.org/pull/16/files#diff-de6dd4b4c889fe0882cfd3f6df5aa451R56). Updating now!

Comment (jairideout) [reply urls.py]: Good catch, thanks @thermokarst!

**Review by jairideout — APPROVED** [review]
Thanks @thermokarst!

**Pull request merged**